frontend/src/locales/en.yml

```diff
@@ -1,94 +1,81 @@
 details:
-  address: 'Address'
-  ceo: 'CEO'
-  company_information: 'Company Information'
-  company_name: 'Company Name'
-  country: 'Country'
-  description: 'Description'
-  exchange: 'Exchange'
-  full_time_employees: 'Full Time Employees'
-  industry: 'Industry'
-  ipo_date: 'IPO Date'
-  isin: 'ISIN'
-  sector: 'Sector'
-  symbol: 'Symbol'
-  website: 'Website'
-
-watchlists:
-  headline: Watchlists
-  updated: Updated
-  show_more: 'Show More'
-
-
+  address: Address
+  ceo: CEO
+  company_information: Company Information
+  company_name: Company Name
+  country: Country
+  description: Description
+  exchange: Exchange
+  full_time_employees: Full Time Employees
+  industry: Industry
+  ipo_date: IPO Date
+  isin: ISIN
+  sector: Sector
+  symbol: Symbol
+  website: Website
+holdings:
+  headline: Holdings
+  table:
+    closeAvgPrice: AvgPrice
+    closeQty: Closed Qty
+    closeTotalPrice: Closed value
+    closedPL: Unrealized P/L
+    currentAvgPrice: Current Price
+    currentPrice: Current Price
+    currentTotalPrice: Current Value
+    dayPL: Day P/L
+    dayPLChange: Daily Change
+    firstBuyAt: First purchace
+    firstSellAt: First sell
+    lastBuyAt: Last purchace
+    lastSellAt: Last sell
+    name: Symbol
+    openAvgPrice: Avg Price
+    openPL: Unrealized P/L
```

Comment:
I was thinking to rather call `P/L` something like `Change`, so its easier to understand. I had never heard of the abbreviation and so might some other less experienced users.
So instead of `Unrealized P/L` it could say `Total Change` and instead of `Day P/L` it could say `Daily Change`.

Reply:
I don't think I added it yet, but we will need to display the Open PL and Closed PL separately, otherwise some  calculations doesn't make any sense.

Ex:
- buy 10 @ 5$ 
- sell 5 @10$
- buy 10 @5$

- current price @11$

Open PL :  90$, closed PL: 50$

---

Not displaying the Closed PL toghether with the remaining open positions is a problem because everyone want to see how profitable a position had been over time.

The "closed holdings" table only display holdings that have no open positions. (Aka historical/past holdings) 

---

I also did some tests to see if combining Open+Close would be viable, but no. ( I got a spreadsheet with theses tests I can send you if you want to try by yourself)

Reply:
I replaced `Realized` / `Unrealized`   with `Open`/`Close` and updated the `tableLayout` for both Active and Previous holdings.